_automation/main.go

```diff
@@ -389,6 +432,165 @@ func (s *UpdateService) downloadTypescript(ctx context.Context, g *Grammar) {
 	}
 }
 
+// Add this function to your UpdateService methods
+func (s *UpdateService) downloadSwift(ctx context.Context, g *Grammar) {
+	logger := getLogger(ctx).With("language", "swift")
+	logger.Info("downloading Swift parser")
+
+	// Fetch the latest release information
+	//latestReleaseURL := fmt.Sprintf("%s/releases/latest", g.URL)
+	tag, _, err := fetchLatestReleaseTagAndRev(g.URL)
```

Comment:
we should use `g.Revision` defined in `grammars.json` here instead of fetching the latest version.